.vscode/tasks.json

```diff
@@ -96,150 +84,152 @@
             "group": {
                 "kind": "test",
                 "isDefault": true
             },
             "problemMatcher": []
-        },
+        },        
         {
-            "label": "Install app loading requirements (macOS)",
+            "label": "[setup] Install vscode requirements",
             "type": "shell",
-            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
-            "command": "[ -n '$VIRTUAL_ENV' ] || if ! python3 -m virtualenv --version >/dev/null 2>&1; then python3 -m pip install virtualenv; fi && [ -d 'ledger' ] || python3 -m virtualenv ledger && source ledger/bin/activate && python3 -m pip show ledgerblue >/dev/null 2>&1 || python3 -m pip install ledgerblue",
+            // Installs functional tests python requirements in the docker container. 
+            "command":"docker exec -it ${config:container_name} bash -c 'export BOLOS_SDK=$(echo ${input:sdk}) && make configure'",
             "group": {
                 "kind": "test",
                 "isDefault": true
             },
             "problemMatcher": []
         },
         {
-            "label": "Install app loading requirements (Windows with PowerShell)",
+            "label": "[setup] Install tests requirements",
             "type": "shell",
-            // Checks that virtual env is installed, otherwise installs it. Then installs ledgerblue in a virtualenv.
-            "command": "cmd.exe /C 'if not exist ledger (python -m pip install virtualenv && python -m venv ledger && call ledger\\Scripts\\activate.bat && python -m pip install ledgerblue)'",
+            // Installs functional tests python requirements in the docker container. 
+            "command":"docker exec -it -u 0 ${config:container_name} bash -c 'apk add gcc musl-dev python3-dev && pip install -r tests/requirements.txt'",
             "group": {
                 "kind": "test",
                 "isDefault": true
             },
             "problemMatcher": []
-        },
+        }
     ],
     "inputs": [
         {
-            "id" : "sdk",
-            "type": "pickString",
-            "description": "Choose a SDK to build with",
-            "options": [
-                "$NANOS_SDK",
-                "$NANOX_SDK",
-                "$NANOSP_SDK",
-                "$STAX_SDK",
-            ]
-        },
-        {
-            "id" : "model",
-            "type": "pickString",
-            "description": "Which model to run speculos for ?",
-            "options": [
-                "nanos",
-                "nanox",
-                "nanosp",
-                "stax",
-            ]
+            "id": "model",
+            "type": "command",
+            "command": "shellCommand.execute",
+            "args": {
+                
+                "command" : "python ${workspaceFolder}/.vscode/config_picker.py ${workspaceFolder}/.vscode/targets.json model ${command:cpptools.activeConfigName}",
```

Comment:
if I understand well here we need to have python to build. We should avoid to have this dependency.

Reply:
I tried with a shell script too, but it was linux only, python was easy scripting cross-platform, but open to other ideas for implementation (wanted to avoid having to pick SDK and target names separately)

Reply:
I agree that python is cross-platform and easy to use but we have to keep in mind that we provide a docker which needs to be self-sufficient to build an app without adding any extra dependency outside the docker...

Reply:
Did a fix to rely on another vscode extension that avoids needing python